src/main/java/commands/inputParser.java

```diff
@@ -0,0 +1,115 @@
+package commands;
+
+import exceptions.AragornException;
+
+public class inputParser {
+    private String[] splitInput = new String[3];
+
+    public inputParser(String userInput, String commandType) throws AragornException {
+        String[] splitDeadline;
+        String[] splitEvent;
+        String LINE =  "    __________________________________________________________\n";
+        switch (commandType) {
+            case "MARK":
+                this.splitInput[0] = String.valueOf(Integer.parseInt(userInput.substring(5).trim()) - 1);
+                this.splitInput[1] = null;
+                this.splitInput[2] = null;
+                break;
+
+            case "UNMARK":
+                this.splitInput[0] = String.valueOf(Integer.parseInt(userInput.substring(7).trim()) - 1);
+                this.splitInput[1] = null;
+                this.splitInput[2] = null;
+                break;
+
+            case "TODO":
+                try {
+                    this.splitInput[0] = userInput.substring(4);
+                    if (this.splitInput[0].trim().isEmpty()){
+                        throw new AragornException(LINE + "    Task description is empty!\n" + LINE);
+                    }
+                    this.splitInput[1] = null;
+                    this.splitInput[2] = null;
+                } catch (AragornException e) {
+                    System.out.println(e.getMessage());
+                }
+                break;
+
+            case "DEADLINE":
```

Comment:
the command types can perhaps be stored as constants or you could probably use enums to avoid the usage of magic literals.